### PR TITLE
A function to retrieve Github Releases of taxonomic resources

### PR DESCRIPTION
**Pull request by fontikar**
Addressing: https://github.com/traitecoevo/APCalign/discussions/243#discussioncomment-11192648

### CODE REVIEW

File: R/load_taxonomic_resources.R [hunk 1]
```diff
@@ -487,3 +487,45 @@ dataset_get <- function(version = default_version(),
...
+    
+    if(httr::http_error(response) | !network){
+      message("API currently down, try again later")
+      return(invisible(NULL))
+    } else
+      release_data <- httr::content(response, "text") |> jsonlite::fromJSON()
+    
+    # Create table
+    dplyr::tibble(versions = unique(release_data$tag_name) |> sort(decreasing = TRUE)) |> 
+      dplyr::filter(!versions == "2020-05-14")
```
Comment (dfalster) [inline R/load_taxonomic_resources.R:529]: Can we add a content why this excluded?

File: vignettes/APCalign.Rmd [hunk 2]
```diff
@@ -9,77 +9,102 @@ vignette: >
...
 
 -   Accessing up-to-date taxonomic information from the [Australian Plant Census](https://biodiversity.org.au/nsl/services/search/taxonomy) and the [Australia Plant Name Index](https://biodiversity.org.au/nsl/services/search/names).
 -   Aligning authoritative names to your taxonomic names using our [fuzzy matching algorithm](https://traitecoevo.github.io/APCalign/articles/updating-taxon-names.html)
 -   Updating your taxonomic names in a transparent, reproducible manner
 
 ## Installation
 
-```r
-install.packages("remotes")
+'APCalign' is currently not on CRAN. You can install its current developmental version using
```
Comment (dfalster) [inline vignettes/APCalign.Rmd:20]: What is needed to submit to CRAN?

**Review by dfalster — DISMISSED** [review]

File: vignettes/APCalign.Rmd (hunk 2 above)
Comment (wcornwell) [reply vignettes/APCalign.Rmd:20]: it is on CRAN isn't it?

Comment (wcornwell) [reply vignettes/APCalign.Rmd:20]: we're on CRAN but failing a couple checks: https://cran.r-project.org/web/checks/check_results_APCalign.html I think this PR might fix that though

**Comment by wcornwell** [comment]
remember to bump version number in the description

File: vignettes/APCalign.Rmd (hunk 2 above)
Comment (fontikar) [reply vignettes/APCalign.Rmd:20]: I'm on it!!

**Review by wcornwell — DISMISSED** [review]

**Review by wcornwell — APPROVED** [review]

**Pull request merged**